src/python/pants/backend/terraform/tool.py

```diff
@@ -46,6 +46,50 @@ class TerraformTool(TemplatedExternalTool):
     @classproperty
     def default_known_versions(cls):
         return [
+            "1.9.0|linux_arm64|f5c0a49b482c008a6afd2248c08ca919e599c1154a850ff94809f4a85c86eb3b|24710285",
```

Comment:
I'd also like to bump the default version to 1.9.0, thoughts? Anyone using <1.8 should be doing to deliberately.